btcec/internal_test.go

```diff
@@ -69,3 +69,9 @@ func (curve *KoblitzCurve) TstDoubleJacobian(x1, y1, z1, x3, y3, z3 *fieldVal) {
 func NewFieldVal() *fieldVal {
 	return new(fieldVal)
 }
+
+// NonceRFC6979 generates a ECDSA nonce (`k`) deterministically according to RFC 6979.
```

Comment:
`TstNonceRFC6979 makes the nonceRFC6979 function available to the test package.`